cmd/tk/workflow.go

```diff
@@ -25,12 +25,14 @@ const (
 )
 
 type workflowFlagVars struct {
-	targets []string
+	targets     []string
+	applyLabels *bool
 }
 
 func workflowFlags(fs *pflag.FlagSet) *workflowFlagVars {
 	v := workflowFlagVars{}
 	fs.StringSliceVarP(&v.targets, "target", "t", nil, "only use the specified objects (Format: <type>/<name>)")
+	v.applyLabels = fs.BoolP("label", "l", false, "Apply tanka metadata labels to all resources")
```

Comment:
having this as a feature flag sounds dangerous – people will forget to pass it, resulting in unset labels for subsets of resources. I see two options here:

1. Just do it. While it will cause changes for every environment, these are purely informative.
2. Feature-flag it in `spec.json`. See my original pruning PR for that: https://github.com/grafana/tanka/blob/2f61573f7d8df6ec2dff6b3dbf3da8a09ce12eb4/pkg/spec/v1alpha1/config.go#L47

Reply:
I'm gonna go with #1. I did intend to switch from opt-in to opt-out at some point in the future, but honestly this is either something that should be done or it isn't.